tests/api/test_messages.py

```diff
@@ -1,11 +1,19 @@
 import itertools
-from typing import Dict, Iterable
+from typing import Dict, Iterable, List, Optional, Callable, Union
 
+import aiohttp
 import pytest
 
 from .utils import get_messages_by_keys
 
 MESSAGES_URI = "/api/v0/messages.json"
+MESSAGES_PAGE_URI = "/api/v0/messages/page/{page}.json"
+
+
+def get_messages_by_predicate(
```

Comment:
This function seems to add more overhead than it solves 🤔. 
I also feel like I commenting on this exact code before 🤔🤔.

What about using https://docs.python.org/3/library/functions.html#filter instead ? It seems appears to do the exact same thing.

Reply:
Yeah you're not wrong. I'll remove it.